src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -915,6 +911,21 @@ public function insert_posts( $data = array() ) {
 	}
 
 	/**
+	 * Adds the import record and origin to the imported event
+	 *
+	 * @param int $id Event ID
+	 * @param int $record_id Import Record ID
+	 * @param string $origin Import Origin
+	 */
+	public static function add_record_to_event( $id, $record_id, $origin ) {
```

Comment:
Can we place this on the `Records` class to avoid needing to use a Static method here on an Abstract?